NitroxClient/GameLogic/RemotePlayer.cs

```diff
@@ -284,7 +290,7 @@ private void UpdateEquipmentVisibility()
         }
         
         /// <summary>
-        /// Allows the remote player model to have its lightings dynamicly adjusted
+        /// Allows the remote player model to have its lightnings dynamically adjusted
```

Comment:
Not sure why you talk about lightnings, since the SkyApplier is meant for lighting